From 04c24c6ecc3ed25dd72ee78cb331a977a98c70de Mon Sep 17 00:00:00 2001
From: Hardik T Shah <hardik.t.shah@intel.com>
Date: Thu, 10 Mar 2016 11:32:51 +0530
Subject: [PATCH 0138/1240] ASoC: Intel: Skylake: Interface change between
 firmware and driver.

Copier interface changed between the firmware and driver.
This patch takes care of the firmware interface change.

Change-Id: I475cde41a4a008808cf7d88fadc20639879fbff4
Signed-off-by: Hardik T Shah <hardik.t.shah@intel.com>
---
 sound/soc/intel/skylake/skl-topology.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/intel/skylake/skl-topology.h b/sound/soc/intel/skylake/skl-topology.h
index 9102c17..e4173f7 100644
--- a/sound/soc/intel/skylake/skl-topology.h
+++ b/sound/soc/intel/skylake/skl-topology.h
@@ -199,8 +199,8 @@ union skl_connector_node_id {
 	u32 val;
 	struct {
 		u32 vindex:8;
-		u32 dma_type:4;
-		u32 rsvd:20;
+		u32 dma_type:5;
+		u32 rsvd:19;
 	} node;
 };
 
-- 
2.7.4

